BaragonUI/app/components/status/StatusPage.jsx

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import classNames from 'classnames';
-import { Link } from 'react-router';
+import { browserHistory } from 'react-router';
 import rootComponent from '../../rootComponent';
 import { refresh } from '../../actions/ui/status';
 
-const StatusPage = (props) => {
+import WorkerStatus from './WorkerStatus';
+import PendingRequests from './PendingRequests';
+import RequestSearch from './RequestSearch';
+
+const navigateToRequest = (requestId) => {
+  browserHistory.push(`/requests/${requestId}`);
```

Comment:
there is an easier way to do this one that uses the react router. We have it set up so that you can have something like

```
import { withRouter } from 'react-router';
...
router.push(`/requests/${requestId}`);
...
export default withRouter(connect((state) => ({
   status: state.api.status.data,
   workers: state.api.workers.data,
   queuedRequests: state.api.queuedRequests.data
 })))(rootComponent(StatusPage, refresh));